src/libOpenImageIO/color_ocio.cpp

```diff
@@ -1997,6 +1997,170 @@ ColorConfig::parseColorSpaceFromString(string_view str) const
 }
 
 
+//////////////////////////////////////////////////////////////////////////
+//
+// Color Interop ID
+
+namespace {
+enum class CICPPrimaries : int {
```

Comment:
I think we should have enums for Unspecified = 2 for  CICPPrimaries and CICPTransfer, too; and "Unknown" should map to 2-2-2. 

I also think, for the time being, we should have CICP 1,2 also map to "g24_rec709_display" (but not necessarily the other way around).